website/client-old/js/controllers/notificationCtrl.js

```diff
@@ -62,6 +69,7 @@ habitrpg.controller('NotificationCtrl',
 
           $scope.ageDailies = function () {
             User.runCron();
+            isRunningYesterdailies = false;
```

Comment:
.then here too? Also what happens if the modal is closed without starting the day? isRunningYesterdailies will never reset: not sure if it's possible though